libraries/vdf/src/sortition.cpp

```diff
@@ -10,7 +10,10 @@ VdfSortition::VdfSortition(SortitionParams const& config, vrf_sk_t const& sk, by
   difficulty_ = calculateDifficulty(config);
 }
 
-bool VdfSortition::isOmitVdf(SortitionParams const& config) const { return threshold <= config.vrf.threshold_lower; }
+bool VdfSortition::isOmitVdf(SortitionParams const& config) const {
+  return config.vrf.threshold_upper >= config.vrf.threshold_range &&
```

Comment:
I guess that one should be easier to understand 
```threshold <= std::max(config.vrf.threshold_upper - config.vrf.threshold_range, 0)```

Reply:
Both of these values are unsigned integers. Does this still work even if threshold_upper is smaller than threshold_range?

Reply:
I dont sure it is defined behaviour, but I test it and it was negative. But we can also cast one of that values to int32_t for example and expect it to be valid  in any situation

Reply:
Your solution is shorter and more readable but I think that casting to int_32 is less efficient. Maybe @MatusKysel or @JakubFornadel give their opinion on this

Reply:
LGTM as is